dyno/panda/dyno_panel.py

```diff
@@ -124,15 +182,47 @@ def writeFile():
 
 #Create the start/stop button
 start_img = ImageTk.PhotoImage(Image.open(start_path).resize([70, 70]))
-button = tk.Button(bg = color_bg, activebackground = color_bg, text = "Pause", image = start_img, borderwidth = 0)
+button = tk.Button(bg = color_bg, activebackground = color_bg, text = "Run", image = start_img, borderwidth = 0)
 button.place(relx = 0.5, rely = 0.35, anchor='center')
 button.bind("<Button-1>", handle_click)
 #---------------------
 
 #Create labels to display sensor readings
 for i in range(int(num_vars)): #First Column
-    label1 = tk.Label(window, textvariable=tk_stringvars[i], bg=color_bg, fg=color_label, font=font_text)
-    label1.place(relx = LABEL_COL1_PLACE['relx'], rely = LABEL_COL1_PLACE['rely']+i*LABEL_SPACE, anchor='w')
+    data_label = tk.Label(window, textvariable=tk_stringvars[i], bg=color_bg, fg=color_label, font=font_text)
+    data_label.place(relx = LABEL_COL1_PLACE['relx'], rely = LABEL_COL1_PLACE['rely']+i*LABEL_SPACE, anchor='w')
+#---------------------
+
+#Create entry fields to get magnet values
+magnet_canvas = tk.Canvas(window, width=150, height=150, bg=color_bg)
+magnet_canvas.place(relx=0.75, rely=0.55, anchor="center")
+magnet_title = tk.Label(window, text="Magnets", bg=color_bg, fg=color_fg, font=font_magnet)
+magnet_canvas.create_window(75, 0, window=magnet_title)
```

Comment:
Can we use constants for the parameters? I've seen 75 and x+75 in a couple of places but it's not too clear to me what that number represents.

Reply:
Those values are used to place the components in the window. I went with relx and rely (as percentages) to make the box resize nicely if you drag the edges. Unfortunately I dont really see it being helpful to have constants since the values just happen to be reused by coincidence